dubbo-registry/dubbo-registry-nacos/pom.xml

```diff
@@ -27,7 +27,7 @@
     <artifactId>dubbo-registry-nacos</artifactId>
 
     <properties>
-        <nacos.version>0.6.2</nacos.version>
+        <nacos.version>2.0.0</nacos.version>
```

Comment:
Hi, don't modify the nacos-client version. 2.6.x is too old, if someone need use nacos-client 2.x, it should override by selfs.

Reply:
OK